### DISCUSSION TITLE
Multi-seat booking placeholders

### DISCUSSION THREAD

**Discussion by markrdavidson** [liedekef/events-made-easy#21] [Q&A] [answered]
Hi Franky,

I am able to use the placeholder #_MULTIBOOKING_TOTALPRICE ... but this doesn't include the VAT.

Is it possible to add a #_MULTIBOOKING_TOTALPRICE_VAT_ONLY placeholder in a future release?  Or am I missing a trick?  If I use #_TOTALPRICE_VAT_ONLY, it only shows the VAT on a single item in the multi-seat booking.

Thanks,
Mark

**Comment by liedekef** [comment]
I think you mean that #_MULTIBOOKING_TOTALPRICE includes the vat, but #_MULTIBOOKING_TOTALPRICE_VAT_ONLY doesn't exist, correct?

**Reply by markrdavidson** [reply]
#_MULTIBOOKING_TOTALPRICE does *not* appear to include VAT.

#_MULTIBOOKING_TOTALPRICE_VAT_ONLY does not appear to exist.

**Comment by liedekef** [comment, marked as answer]
#_MULTIBOOKING_TOTALPRICE just calls eme_get_total_booking_price() for each booking and adds them up, while #_TOTALPRICE just calls eme_get_total_booking_price() for 1 specific booking. So it does include VAT.

But this should help: https://github.com/liedekef/events-made-easy/commit/c85e7be8bf16ad217c3e7c27cb2e17ac872db04e

Please do be aware that feature requests normally need to be asked via the "Issues" form (github style) :-)
